source/pymongo-async-migration.txt

```diff
@@ -0,0 +1,339 @@
+.. _pymongo-async-migration:
+
+===================================
+Migrate to the {+driver-async+} Driver
+===================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: motor, async, refactor, migration
+
+Overview
+--------
+
+The {+driver-async+} driver is a unification of {+driver-short+} and the `Motor
+library <https://www.mongodb.com/docs/drivers/motor/>`__. In this guide, you can
+identify the changes you must make to migrate an application from
+{+driver-short+} or Motor to the {+driver-async+} driver.
+
+Migrate From {+driver-short+}
+--------------------
+
+The {+driver-async+} driver behaves similarly to {+driver-short+}, but
+all methods that perform network operations are coroutines and must be awaited.
+To migrate from {+driver-short+} to {+driver-async+}, you must update your code
+to replace all uses of ``MongoClient`` with ``AsyncMongoClient`` and add the
+``await`` keyword to all asynchronous method calls. If an asynchronous method is
+called within a function, you must also mark the function as ``async``.
```

Comment:
S: convert to a list: eg

.. you must update your code in the following ways:

- Replace all uses of ...
- Add the ``await`` keyword to..
- ....